ci/do_ci.sh

```diff
@@ -51,21 +48,17 @@ export BAZEL_EXTRA_TEST_OPTIONS="--test_env=ENVOY_IP_TEST_VERSIONS=v4only --test
 export ENVOY_CONTRIB_BUILD_TARGET="//source/exe:envoy-static"
 export ENVOY_CONTRIB_BUILD_DEBUG_INFORMATION="//source/exe:envoy-static.dwp"
 
-# sudo apt-get install google-perftools -y
-# export PPROF_PATH=$(which google-pprof)
-
-if [ "x${BUILD_TYPE:-}" != "x" ] ; then
+if [ "${BUILD_TYPE:-}" != "" ] ; then
   BUILD_CONFIG="--config=$BUILD_TYPE"
 fi
 echo "$BUILD_CONFIG is ${BUILD_CONFIG}"
 
-echo "test $BUILD_CONFIG" >> "${SOURCE_DIR}/.bazelrc"
+echo "test $BUILD_CONFIG" >> "${SOURCE_DIR}/test.bazelrc"
 
 echo Building
-sed -i "s|//test/tools/schema_validator:schema_validator_tool|@envoy//test/tools/schema_validator:schema_validator_tool|" "$UPSTREAM_ENVOY_SRCDIR/ci/do_ci.sh"
-sed -i "s|bazel-bin/test/tools/schema_validator/schema_validator_tool|bazel-bin/external/envoy/test/tools/schema_validator/schema_validator_tool|" "$UPSTREAM_ENVOY_SRCDIR/ci/do_ci.sh"
-sed -i "s|VERSION.txt|ci/FAKEVERSION.txt|" "$UPSTREAM_ENVOY_SRCDIR/ci/do_ci.sh"
-sed -i "s|\${ENVOY_SRCDIR}/VERSION.txt|ci/FAKEVERSION.txt|" "$UPSTREAM_ENVOY_SRCDIR/ci/build_setup.sh"
+sed -i 's|"//contrib/..."||' "$UPSTREAM_ENVOY_SRCDIR/ci/do_ci.sh"
+sed -i 's|//distribution|@envoy//distribution|' "$UPSTREAM_ENVOY_SRCDIR/ci/do_ci.sh"
+sed -i 's|bazel-bin/distribution|bazel-bin/external/envoy/distribution|' "$UPSTREAM_ENVOY_SRCDIR/ci/do_ci.sh"
```

Comment:
It's possible that this causes envoy to be built without our wrapper, which would clearly be not what we want